gui/src/widgets/popupwidget.cpp

```diff
@@ -0,0 +1,95 @@
+#include "widgets/popupwidget.h"
+#include "stylehelper.h"
+#include "ui_popupwidget.h"
+
+
+using namespace scopy::gui;
+
+PopupWidget::PopupWidget(QWidget* parent) :
+	ui(new Ui::PopupWidget),
+	QWidget(parent)
+{
+	ui->setupUi(this);
+
+	StyleHelper::initColorMap();
+	StyleHelper::BlueButton(ui->btnContinue, "btnContinue");
+	StyleHelper::BlueButton(ui->btnExit, "btnExit");
+	StyleHelper::OverlayMenu(this, "popupOverlay");
+	ui->title->setStyleSheet("background-color: #101014; border: 1px solid #070708"); // color match the description QTextBrowser
```

Comment:
This code is duplicated. Why do you need 2 constructors ? Can you just create a constructor and a setDescription method ?